### ISSUE TITLE
ilsvrc.py uses cv2.imread to read image. The preprocessing assumes images are in rgb.

### ISSUE THREAD

**Issue by EchoWho** [ppwwyyxx/tensorpack#176]
In imagenet-resnet, the hard-coded augmentation for training images are for rbg images (from fb.resnet.torch and their torch/images https://github.com/torch/image/blob/master/doc/saveload.md). in ilsvrc.py, your images are generated from cv2.imread, which gave you bgr images (http://docs.opencv.org/3.0-beta/doc/py_tutorials/py_gui/py_image_display/py_image_display.html). 

It should be a minor issue for neural networks, and those mean/std/pca were close any way.

**Comment by ppwwyyxx** [comment]
You're right! Thanks!
However, changing the code may affect the claimed accuracy of my pre-trained models (and I don't want to train again), so I'll run some tests before fixing this.
